client/src/actions/map.js

```diff
@@ -9,7 +11,7 @@ function fetchMonumentsPending() {
 function fetchMonumentsSuccess(monuments) {
     return {
         type: FETCH_MAP_MONUMENTS_SUCCESS,
-        payload: monuments
+        payload: {monuments}
```

Comment:
Any reason this payload is in curly braces and the one for `searchMonumentsSuccess` isn't?

Reply:
I will double check why they're different, but basically I was running into issues where for example `state.searchPage` would be the search results array instead of an object with `monuments: []` so this fixed that. Need to double check if there is a discrepancy or an actual reason why they're different